models/lfs_lock.go

```diff
@@ -71,15 +74,15 @@ func CreateLFSLock(lock *LFSLock) (*LFSLock, error) {
 }
 
 // GetLFSLock returns release by given path.
-func GetLFSLock(repoID int64, path string) (*LFSLock, error) {
+func GetLFSLock(repo *Repository, path string) (*LFSLock, error) {
```

Comment:
Why this change? Is it really worth passing the repository if the only thing we care about it the id?

Reply:
Now I remember it is to use HasAccess that need a repo reference.

Reply:
Maybe I could make a func accessLevel and hasAccess that use repoID more generally for better perf ? (I think in a other PR ?)

Reply:
Ah, I must have missed the call to `CheckLFSAccessForRepo`. Yes, perhaps in another PR

Reply:
In fact after reviewing code of accessLevel, it use repo.IsPrivate that need a repo reference so.